pkgs/tools/misc/migra/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+}: let
+  pname = "migra";
+  version = "3.0.1647431138";
+in
+  python3.pkgs.buildPythonApplication rec {
+    inherit pname version;
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "sha256-VddI+dLgOz2Fjvj8jRD/rvwTf1HADalFlDOb/xCxtfg=";
+    };
+
+    checkInputs = [python3.pkgs.pytestCheckHook];
+    pytestFlagsArray = [
+      "-x"
+      "-svv"
+      "tests"
+    ];
+
+    propagatedBuildInputs = [
+      python3.pkgs.schemainspect
+      python3.pkgs.six
+      python3.pkgs.sqlbag
+    ];
+
+    # FIXME(@Soispha): The test don't work without an extensive
```

Comment:
I've checked the pypi output and the test aren't packed here either. 
I'm relatively sure, that the test aren't packaged for a reason, and I'd suggest just going along with it

Reply:
The fix is commonly to use `fetchFromGitHub` or such instead of `fetchPypi`.